server/planning/assignments/assignments.py

```diff
@@ -402,9 +402,10 @@ def send_assignment_notification(self, updates, original=None, force=False):
                     )
                     event["LOCATION"] = formatted_location
 
-            event["CREATED"] = created
-            event["LAST-MODIFIED"] = updated
-            event["STATUS"] = assigned_to["state"]
+            event.add("CREATED", created)
+            event.add("LAST-MODIFIED", updated)
+            event.add("DTSTAMP", updated)
+            event.add("STATUS", "CANCELED" if assigned_to["state"] == "canceled" else "CONFIRMED")
```

Comment:
It might be good to use `planning.common.ASSIGNMENT_WORKFLOW_STATE` named tuple when checking the state

Reply:
Good point, will do!